packages/console/src/pages/Applications/components/GuideHeader/index.tsx

```diff
@@ -1,27 +1,55 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { useTranslation } from 'react-i18next';
 
+import GetSample from '@/assets/images/get-sample.svg';
 import Button from '@/components/Button';
 import CardTitle from '@/components/CardTitle';
 import DangerousRaw from '@/components/DangerousRaw';
 import IconButton from '@/components/IconButton';
 import Spacer from '@/components/Spacer';
+import Tooltip from '@/components/Tooltip';
 import Close from '@/icons/Close';
+import { SupportedSdk } from '@/types/applications';
 
 import * as styles from './index.module.scss';
 
 type Props = {
   appName: string;
-  selectedSdk: string;
+  selectedSdk: SupportedSdk;
   isCompact?: boolean;
   onClose: () => void;
 };
 
-const onClickFetchSampleProject = (projectName: string) => {
-  const sampleUrl = `https://github.com/logto-io/js/tree/master/packages/${projectName}-sample`;
-  window.open(sampleUrl, '_blank');
+const getSampleProjectUrl = (sdk: SupportedSdk) => {
+  const githubUrlPrefix = 'https://github.com/logto-io';
+
+  switch (sdk) {
+    case SupportedSdk.iOS:
+      return `${githubUrlPrefix}/swift/tree/master/Demos/SwiftUI%20Demo`;
+    case SupportedSdk.Android:
+      return `${githubUrlPrefix}/kotlin/tree/master/android-sample-kotlin`;
+    case SupportedSdk.React:
+      return `${githubUrlPrefix}/js/tree/master/packages/react-sample`;
+    case SupportedSdk.Vue:
+      return `${githubUrlPrefix}/js/tree/master/packages/vue-sample`;
+    case SupportedSdk.Vanilla:
+      return `${githubUrlPrefix}/js/tree/master/packages/browser-sample`;
+    case SupportedSdk.Traditional:
+      return `${githubUrlPrefix}/express-example`;
```

Comment:
will it be better to refactor the `Guides` const to combine this all together? not a blocker.

Reply:
No, the Guides const contains i18n keys, better not mixing these all together.

Reply:
An issue has been added. Let's talk about it later
https://linear.app/silverhand/issue/LOG-3332/combine-guides-const-and-sample-project-switch-case-into-one